### PR TITLE
Ch3 polish

### PR DESCRIPTION
**Pull request by Darrman**
ch3 plot completed, and a few other things polished up
(main thing is adding a stealable red gem to a fighter so there's a bit of money)

### CODE REVIEW

File: Events/Ch22.event [hunk 1]
```diff
@@ -70,32 +70,70 @@ ENDTRAP
...
+TEXTSHOW Ch22AfterPrepScreen
+TEXTEND
+CAM1 [21,11]
+FlashCursor(25,11,60)
+TEXTCONT
+TEXTEND
+CAM1 [24,2]
+FlashCursor(24,2,60)
+TEXTCONT
+TEXTEND
```
Comment (StanHash) [inline Events/Ch22.event]: `REMA; NoFade`?

File: Events/Ch3.event [hunk 2]
```diff
@@ -78,7 +86,27 @@ UnitGet:
...
 ENUN
-Text(PlaceholderText)
+Text(HouseBG,Ch3RecruitingBoo)
+REMA
+NoFade
+ENDA
+
+UpperHouse:
+CHECK_ACTIVE
+CHECK_ACTIVEID Boo
```
Comment (StanHash) [inline Events/Ch3.event]: `CHECK_ACTIVE` is useless here

File: Text/Chapter Text/Chapter 22.txt [hunk 3]
```diff
@@ -1,27 +1,144 @@
...
+...power he's going to take for himself![N]
+And with that power, lay waste to this[AN]
+entire continent! My friends have only[N]
+one mission: to stop his schemes and[AN]
+restore balance to this world, bound[N]
+for ruin! I will fight to preserve the[AN]
+world, if you'll have me along. You[N]
+will? Great, time to pummel Kefka![A][X]
 
 ## ReinforcementsStopped
```
Comment (StanHash) [inline Text/Chapter Text/Chapter 22.txt]: Dangerously generic text definition name

File: Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
-INLINE NewDeathQuoteTable,Character,##UNKNOWN##,Chapter,Event ID,##UNKNOWN##,Text ID,##UNKNOWN##
+0x9ECD4C,Character,##UNKNOWN##,Chapter,Event ID,##UNKNOWN##,Text ID,##UNKNOWN##
```
Comment (StanHash) [inline Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv]: :(

File: Events/Ch3.event [hunk 5]
```diff
@@ -95,7 +123,7 @@ UNIT 0x86 Knight Breguet Level(1, Enemy, 1) [7,6] 0x00 0x00 0x0 0x00 [IronLance]
 UNIT 0x86 Cavalier Breguet Level(3, Enemy, 1) [7,0] 0x00 0x00 0x0 0x00 [IronSword] AttackInRangeAI
 UNIT 0x86 Fighter Breguet Level(3, Enemy, 1) [11,5] 0x00 0x00 0x0 0x00 [IronAxe] PursueWithoutHeedAI
 UNIT 0x86 Fighter Breguet Level(3, Enemy, 1) [15,4] 0x00 0x00 0x0 0x00 [IronAxe] PursueWithoutHeedAI
-UNIT 0x86 Fighter Breguet Level(3, Enemy, 1) [13,7] 0x00 0x00 0x0 0x00 [IronAxe] PursueWithoutHeedAI
+UNIT 0x86 Fighter Breguet Level(3, Enemy, 1) [13,7] 0x00 0x00 0x0 0x00 [IronAxe, RedGem] PursueWithoutHeedAI
```
Comment (Crazycolorz5) [inline Events/Ch3.event]: It's intentional that this is not dropping, right?

File: Mugs/PortraitShiftTable.txt [hunk 6]
```diff
@@ -0,0 +1,122 @@
+//// Portrait Upshift Edit(or?)
```
Comment (Crazycolorz5) [inline Mugs/PortraitShiftTable.txt]: I'm changing a lot of portrait IDs in another branch. Will that cause issues here?

File: Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv (hunk 4 above)
Comment (Crazycolorz5) [reply Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv]: I second that :(.
Why you do dis?

File: Text/battle_quotes.txt [hunk 7]
```diff
@@ -23,3 +23,9 @@ HATE HATE HATE HATE HATE HATE[N]
 HATE HATE HATE HATE HATE HATE[AN]
 HATE HATE HATE HATE HATE HATE[N]
 HATE HATE HATE HATE HATE YOU![A][X]
+
+
+## MealyDeath
+[OpenMidRight][LoadFace][0x01][0x01]
```
Comment (Crazycolorz5) [inline Text/battle_quotes.txt]: Portrait [0x01]? 
use a ParseDefinition definition here, and refer to that.

File: Text/character_text.txt [hunk 8]
```diff
@@ -274,3 +274,33 @@ Nichol[X]
 ## descNichol
 A ruthless man who has two fears:[N]
 Death, and a ruined Christmas.[X]
+
+## Ch3BossName
+Ch. 3 Boss[X]
```
Comment (Crazycolorz5) [inline Text/character_text.txt]: Please flesh out all of these names and descriptions.

File: Mugs/PortraitShiftTable.txt (hunk 6 above)
Comment (Darrman) [reply Mugs/PortraitShiftTable.txt]: I was never able to get it to work. Will remove file.

File: Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv (hunk 4 above)
Comment (Crazycolorz5) [reply Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv]: Please revert this change and send me the `Death Quote Editor.event` that's generated during the MAKE HACK process.

**Pull request merged**